gatling/gatling.py

```diff
@@ -171,9 +171,10 @@ def sink(selector, not_done):
                 if not p:
                     if verbose:
                         running = len(p_filled) + len(p_open)
-                        print(f"# {my_name} STARTED A PROCESS (1 + {running} + {len(res)}):",
-                              *cmd,
-                              file=sys.stderr)
+                        print(
+                            f"# {my_name} STARTED A PROCESS (1 + {running} + {len(res)}):",
+                            *cmd,
+                            file=sys.stderr)
```

Comment:
New version of yapf?

Reply:
Uninstalled and reinstalled, (version 0.29.0), ran make, and see no difference.

Reply:
Did you `make lint`? yapf is not called by default.